themesConfig.json

```diff
@@ -3,6 +3,10 @@
     "items": [
       {
         "url": "http://qwc2.sourcepole.ch/wms/wolfsburg/qwc2_demo",
+        "wmsBasicAuth": {
+          "username": "test",
+          "password": "test"
+        },
```

Comment:
I'd leave this out, since it is kinda misleading as this server does not require authentication.

Reply:
Oh yes, sorry, this was not meant to be included.